test/distributed/org/apache/cassandra/distributed/test/SimpleReadWriteTest.java

```diff
@@ -389,30 +268,4 @@ public void pagingTests() throws Throwable
 
         }
     }
-
-    @Test
-    public void metricsCountQueriesTest() throws Throwable
```

Comment:
please don't delete any tests.  if you want to move them into different files (some code can be extracted some can't) im ok with this, but gripping for `metricsCountQueriesTest` shows no matches other than this removal.